### DISCUSSION TITLE
Any perspective of Avalonia support?

### DISCUSSION THREAD

**Discussion by Tyrrrz** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3199] [Q&A] [answered]
Hello, there!

I've been using this project for nearly 7 years so far. Thank you for all the effort you've put into it 🙂

Recently, I've been meaning to migrate my projects from WPF to Avalonia and was wondering if MDIXAML has Avalonia anywhere on its roadmap? I did a cursory search through issues and discussions and saw no mention of it.

I know about the existence of other MD libraries for Avalonia, namely https://github.com/AvaloniaCommunity/Material.Avalonia, but it's not yet as feature-rich as this project.

**Comment by MichelMichels** [comment, marked as answer]
There hasn't been any work done so far, but I'm aware of the Avalonia project. In following weeks, I'll investigate the possibility of this path.

**Reply by nicolaihenriksen** [reply]
There is already a repository/nuget package out in the wild which is inspired from this repository: [Material.Avalonia](https://github.com/AvaloniaCommunity/Material.Avalonia)
